src/presentation/http/router/note.ts

```diff
@@ -164,14 +164,9 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
     /**
      * Get all tools used in the note
      */
-    const noteToolsNames = new Set<string>();
-
-    note.content.blocks.forEach((block: { type: string }) => {
-      noteToolsNames.add(block.type);
-    });
-
-    const noteTools = await editorToolsService.getToolsByNames(Array.from(noteToolsNames));
+    const noteToolsIds : EditorTool['id'][] = Array.from(note.tools).map((tool) => tool.id);
```

Comment:
why do you we need `Array.from()` here? `note.tools` are already an array aren't they?